packages/form/src/form-item.vue

```diff
@@ -145,7 +146,7 @@ export default defineComponent({
       if (!props.label && !props.labelWidth && isNested.value) {
         return {}
       }
-      const labelWidth = props.labelWidth || elForm.labelWidth
+      const labelWidth = addUnit(props.labelWidth) || addUnit(elForm.labelWidth)
```

Comment:
`addUnit` here will introduce a `warning`, if the `labelWidth` is by default using the value assigned from the `ElForm` component. It's better to first check its existence, then perform the conversion.

Reply:
thank you!

Reply:
If the label-width is not set for the ElFormItem component, a warning will still be triggered，Now the label-width must be set?

Reply:
Is there already a bug report for this?
Working with beta.55 is very difficult with all those error messages showing up...

Reply:
@plehnen #2490

Reply:
@equt: that's the thing which breaks. because `props.labelWidth` doesn't exists (when only `elForm.labelWidth` is used), addUnit will produce a warning: "**ElementPlusError: [Util] binding value must be a string or number**"

Reply:
ah, nevermind. my e-mail client was showing me the comment from above. got me confused.